scripts/ci-entrypoint.sh

```diff
@@ -214,6 +215,22 @@ wait_for_pods() {
     done
 }
 
+# wait_for_extra_nodes accommodates building large clusters gradually
+wait_for_extra_nodes() {
+    local existing_nodes="${WORKER_MACHINE_COUNT}"
+    local remaining_extra_nodes="${EXTRA_NODES}"
+    while [[ "${remaining_extra_nodes}" -gt 10 ]]; do
+        NEW_REPLICA_COUNT=$((existing_nodes + 10))
+        "${KUBECTL}" --kubeconfig "${REPO_ROOT}/${KIND_CLUSTER_NAME}.kubeconfig" scale machinedeployment/"${CLUSTER_NAME}"-md-0 --replicas="${NEW_REPLICA_COUNT}"
+        sleep 120
```

Comment:
Is there a `kubectl wait` command we can use here instead? Going from 0 to 5000 nodes would take over 16 hours this way if my math is right.

Reply:
+1 to both